support/dev-tools/polkadot-utils/src/lib/TxnBatcher.ts

```diff
@@ -0,0 +1,40 @@
+export class TxnBatcher {
+    private isInSend = false;
+    private queued = new Map();
+
+    constructor(private readonly api: any) {
+        this.api = api;
+    }
+
+    public async send(txn: any, signer: any) {
```

Comment:
This doesn't allow me to wait on this transaction being submitted. A user of this class would do something like the following:

```ts
const hash = await batcher.send(txn, signer);
console.log('Transaction submitted');
return {txn: hash};
```